packages/flutter_tools/lib/src/test/runner.dart

```diff
@@ -44,7 +49,7 @@ Future<int> runTests(
   if (machine) {
     testArgs.addAll(<String>['-r', 'json']);
   } else {
-    testArgs.addAll(<String>['-r', 'compact']);
+    testArgs.addAll(<String>['-r', 'expanded']);
```

Comment:
Is this an unrelated change?

Reply:
It is, reverted

Reply:
It was also the source of the test failures